package/gluon-ddhcpd/luasrc/lib/gluon/upgrade/316-ddhcp

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/lua
```

Comment:
There's no need to sort this behind the firewall rules, both can be put at position 315.